pages/care-package/[guid]/review.js

```diff
@@ -15,37 +15,56 @@ const settingsTypes = [
 const ReviewPackageDetailsPage = () => {
   const router = useRouter();
   const carePackageId = router.query.guid;
-
   const { data } = useCarePackageApi.summary(carePackageId);
 
-  const checkSettings = (settings) => settings && settingsTypes.find((item) => settings[item.field] === true)?.text;
+  const [fundedNursingCareClaimCollector] = useState({
+    2: 'Hackney Council (gross/net)',
+    1: 'Supplier (gross/net)',
+  });
+
+  const [careChargesClaimCollector] = useState({
+    2: 'Hackney Council (gross)',
+    1: 'Supplier (net)',
+  });
+
+  const checkSettings = (settings) => settings && settingsTypes
+    .filter((item) => settings[item.field] === true)
```

Comment:
No need to do explicit check on `true`
```suggestion
    .filter((item) => settings[item.field])
```